redis/Redis.php

```diff
@@ -681,7 +681,7 @@ public function subscribe($channels, $callback) {}
      * @param array        $patterns   an array of glob-style patterns to subscribe
      * @param string|array $callback   Either a string or an array with an object and method.
      *                     The callback will get four arguments ($redis, $pattern, $channel, $message)
-     * @param mixed        Any non-null return value in the callback will be returned to the caller
+     *                     Any non-null return value in the callback will be returned to the caller
```

Comment:
This left part looks strange. Could you please review this comment and update it as well?

Reply:
Sure, I will search more to justify the fix

Reply:
Code examples:

- https://github.com/cheprasov/php-redis-client/blob/e4c5912bd587e10f5d9b84c9f80ef4ba6c9258dc/examples/pubsub.php#L111

- https://programming.vip/docs/php-implements-redis-message-publishing-subscription.html

@isfedorov I think this should be kept identical.
Here is the docs source: https://github.com/phpredis/phpredis/blob/306fa25c19d9164a0c3e2dea20ad19e683319a5f/README.markdown#subscribe

Reply:
Ah, ok, I see. But in this case, removing `@param mixed` was wrong. It should be changed to `@return mixed` if I'm not mistaken. `return value: Mixed. Any non-null return value in the callback will be returned to the caller.`

Reply:
Oh okay, will push that

Reply:
Done in 4a02da7a, it makes sense as it is the exact same content psubscribe has